### PR TITLE
feat: drop php 7.4 and add support for 8.2

### PR DESCRIPTION
**Pull request by st3iny**
Also adjusts workflows.

## TODO
- [x] Adjust required workflow checks

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,20 +12,19 @@ jobs:
...
+        nextcloud-version: ${{ matrix.nextcloud-version }}
         php-version: ${{ matrix.php-version }}
-        extensions: ctype,curl,dom,gd,iconv,intl,json,mbstring,openssl,posix,sqlite,xml,zip
-        coverage: xdebug
-    - name: Checkout Nextcloud
-      run: git clone https://github.com/nextcloud/server.git --recursive --depth 1 -b ${{ matrix.nextcloud-version }} nextcloud
-    - name: Install Nextcloud
-      run: php -f nextcloud/occ maintenance:install --database-name oc_autotest --database-user oc_autotest --admin-user admin --admin-pass admin --database sqlite --database-pass=''
+        php-coverage: 'xdebug'
+        patch-php-version-check: ${{ matrix.php-version == '8.2' }}
```
Comment (ChristophWurst) [inline .github/workflows/test.yml:25]: not necessary anymore with server master

**Review by ChristophWurst — APPROVED** [review]
nit: should be a `feat` commit to be listed in the changelog

File: .github/workflows/test.yml (hunk 1 above)
Comment (st3iny) [reply .github/workflows/test.yml]: Done. I also fixed the matrix to not test stable25 against php 8.2.

**Comment by st3iny** [comment]
@ChristophWurst Please adjust workflows and merge. I'm not authorized (yet?!).

**Comment by ChristophWurst** [comment]
try again

**Pull request merged**